pom.xml

```diff
@@ -572,6 +572,13 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-checkstyle-plugin</artifactId>
           <version>${plugin.checkstyle.version}</version>
+          <dependencies>
+            <dependency>
+              <groupId>com.puppycrawl.tools</groupId>
+              <artifactId>checkstyle</artifactId>
+              <version>6.11.1</version>
+            </dependency>
+          </dependencies>
```

Comment:
What's the purpose of this dependency ?

Reply:
I'm using lambda function as well but don't need to upgrade `checkstyle`. Could you please give an example?

Reply:
I tested it and did not need to upgrade checkstyle. I will clear this change.